modules/mobile/config/routes.rb

```diff
@@ -14,6 +14,7 @@
     post '/claim/:id/request-decision', to: 'claims_and_appeals#request_decision'
     get '/disability-rating', to: 'disability_rating#index'
     get '/health/immunizations', to: 'immunizations#index'
+    get '/health/location/:id', to: 'immunizations#location'
```

Comment:
URL looks good, I'd give it its own controller and following Rails conventions it would be the #show action.